internal/backend/gs/gs.go

```diff
@@ -239,7 +244,13 @@ func (be *Backend) Save(ctx context.Context, h restic.Handle, rd restic.RewindRe
 	//
 	// restic typically writes small blobs (4MB-30MB), so the resumable
 	// uploads are not providing significant benefit anyways.
-	w := be.bucket.Object(objName).NewWriter(ctx)
+	var test1 []byte
+	test1, _ = base64.StdEncoding.DecodeString(be.encryptKey)
```

Comment:
Please use a descriptive variable name. What happens if the key cannot be base64-decoded? Just ignoring the error seems to be a bad idea.

Reply:
Renamed and check error.